cli/options/src/lib.rs

```diff
@@ -200,6 +206,10 @@ pub struct Options {
     /// [cargo] caching is disabled by default, this flag enables it back.
     #[arg(long="enable-cargo-cache", action=clap::builder::ArgAction::SetTrue)]
     pub force_cargo_build: ForceCargoBuild,
+
+    /// Apply the command to the dependencies of the primary package
```

Comment:
The description is a bit terse, what about:
```suggestion
    /// By default, the action ([translate], [lint], or [json]) is applied only on the package selected by the user (i.e. the package or virtual workspace of the current directory, or the one selected with [-C -p package ;] or [-C --manifest-path=/some/path/Cargo.toml ;]. With `--deps`, the action will be applied to every local package in the dependency closure.
```